client-java/pom.xml

```diff
@@ -0,0 +1,234 @@
+
+				<!-- encodings -->
+				<encoding>UTF-8</encoding>
+				<project.build.sourceEncoding>${encoding}</project.build.sourceEncoding>
+				<project.reporting.outputEncoding>${encoding}</project.reporting.outputEncoding>
+
+				<!-- Plugins -->
+				<junit5.version>5.7.0</junit5.version>
+
+		</properties>
+
+		<build>
+				<plugins>
+						<plugin>
+								<artifactId>maven-surefire-plugin</artifactId>
+						</plugin>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <!-- Please see https://github.com/immutables/immutables/issues/902 -->
+		        <configuration>
+		          <sourcepath>${project.build.sourceDirectory}:${project.build.directory}/generated-sources/annotations</sourcepath>
+		          <additionalDependencies>
+		            <additionalDependency>
+		              <groupId>javax.annotation</groupId>
+		              <artifactId>javax.annotation-api</artifactId>
+		              <version>1.3.2</version>
+		            </additionalDependency>
+		          </additionalDependencies>
+            <additionalOptions>-html5</additionalOptions>
+            <doclint>none</doclint>
+          </configuration>
+      </plugin>
+      <!--
+						<plugin>
+						    <groupId>org.assertj</groupId>
+						    <artifactId>assertj-assertions-generator-maven-plugin</artifactId>
+						    <version>2.1.0</version>
+						    <executions>
+						        <execution>
+						            <goals>
+						                <goal>generate-assertions</goal>
+						            </goals>
+						        </execution>
+						    </executions>
+						    <configuration>
+						        <packages>
+						            <param>uk.gov.gchq.palisade</param>
+						        </packages>
+              <includes>
+                <param>uk\.gov\.gchq\.palisade\.client\.java\..*\.I.*</param>
+              </includes>
+              <generateAssertionsInPackage>uk.gov.gchq.palisade.client.java.assertions</generateAssertionsInPackage>
+              <targetDir>target/generated-test-sources/test-annotations</targetDir>
+														<generateAssertions>false</generateAssertions>
+														<generateBddAssertions>false</generateBddAssertions>
+														<generateSoftAssertions>false</generateSoftAssertions>
+														<generateJUnitSoftAssertions>false</generateJUnitSoftAssertions>          </configuration>
+						</plugin>
+						-->
+				</plugins>
+		</build>
+
+		<dependencies>
+
+				<!-- MAIN -->
+
+				<!-- https://mvnrepository.com/artifact/io.micronaut/micronaut-inject-java -->
+				<dependency>
+						<groupId>io.micronaut</groupId>
+						<artifactId>micronaut-inject-java</artifactId>
+						<version>2.1.0</version>
+				</dependency>
+				<dependency>
+		    <groupId>io.micronaut</groupId>
+		    <artifactId>micronaut-http-client</artifactId>
+      <version>2.1.0</version>
+				</dependency>
+
+				<!-- Palisade -->
+
+				<dependency>
```

Comment:
Please double check, but I don't think that common and readers-common are needed here as dependencies

Reply:
They've now gone :+1: